src/api/posts/src/storage.js

```diff
@@ -22,6 +22,26 @@ const createInvalidFeedKey = (id) => createFeedKey(id).concat(invalidSuffix);
 // "NirlSYranl" to "t:feed:NirlSYranl:delayed"
 const createDelayedFeedKey = (id) => createFeedKey(id).concat(delayedSuffix);
 
+const getInvalidKeys = async () => {
+  let keysArray = [];
+  const stream = redis.scanStream({
+    match: 't:feed:*:invalid',
+  });
+  return new Promise((res, rej) => {
```

Comment:
Prefer `resolve` and `reject` vs. `res`, since it also could mean `response`.  It's better to be clear.